src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -625,6 +625,14 @@ private function isAllowedProperty(string $class, string $property, bool $writeA
                 return false;
             }
 
+            if (\PHP_VERSION_ID >= 80400
+                    && $writeAccessRequired
+                    && $reflectionProperty->isVirtual()
```

Comment:
now I'm wondering: why excluding virtual properties?

Reply:
A backed property can be set even there is no set hook.